docs/docusaurus.config.js

```diff
@@ -29,7 +29,7 @@ const config = {
           remarkPlugins: [math, require('mdx-mermaid')],
           rehypePlugins: [katex],
           sidebarPath: require.resolve('./sidebars.js'),
-          editUrl: 'https://github.com/ingonyama-zk/icicle/tree/main',
+          editUrl: 'https://github.com/ingonyama-zk/icicle/tree/main/docs',
```

Comment:
what does this mean?

Reply:
Its the base URL that opens when clicking on `Edit this page`
![image](https://github.com/user-attachments/assets/33d17035-cd23-4c80-809e-31c3558599e6)